src/main/resources/plugin.properties

```diff
@@ -0,0 +1,10 @@
+id=http-redirect
+name=HTTP Redirect
+version=${project.version}
+description=${project.description}
+class=io.gravitee.policy.http.redirect.HTTPRedirectPolicy
+type=policy
+category=others
+icon=http-redirect.svg
```

Comment:
It's possible to specify the documentation as the icon now. It's even recommended to do :) 

```
documentation=README.adoc
schema=schema-form.json
```